components/elm/cime_config/config_compsets.xml

```diff
@@ -758,7 +758,8 @@
 
   <compset>
     <alias>IGELM_MLI</alias>
-    <lname>2000_DATM%QIA_ELM%SP_SICE_SOCN_MOSART_MALI%SIA_SWAV</lname>
+    <!--lname>2000_DATM%QIA_ELM%SP_SICE_SOCN_MOSART_MALI%SIA_SWAV</lname-->
+    <lname>2000_DATM%QIA_ELM%SP_SICE_SOCN_MOSART_MALI_SWAV</lname>
```

Comment:
This means the MALI component will be active in the build, correct? I can't remember what we decided about that. For now, I'm happy either way. If it's not a pain for any reason, let's leave it like this so the build gets exercised automatically when we run this compset.

Reply:
Yes, the MALI build will be active for this compset, but not the one with %SIA. Having the two compsets allows us to control the build that way